src/py42/services/preservationdata.py

```diff
@@ -19,8 +19,8 @@ def find_file_version(self, file_md5, file_sha256, paths):
         uri = u"/api/v1/FindAvailableVersion"
         return self._connection.post(uri, json=data)
 
-    def get_file_version_list(self, deviceGuid, file_md5, file_sha256, path):
-        params = u"fileSHA256={}&fileMD5={}&deviceGuid={}&path={}"
-        params = params.format(file_sha256, file_md5, deviceGuid, quote(path))
-        uri = u"/api/v1/FileVersionListing?{}".format(params)
+    def get_file_version_list(self, deviceUid, file_md5, file_sha256, path):
```

Comment:
`deviceUid` should be in `snake_case`?

Reply:
I am guessing it is not a breaking change? Or is it? Because it also is being renamed already from `deviceGuid` to `deviceUid`, so we should fix the casing or let it be?

Reply:
I don't expect that anyone is realistically calling this function directly and even if they are its even less likely they're doing it with a named param. We should use the proper casing, thanks.